components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/CmdUtils.java

```diff
@@ -442,6 +583,16 @@ public static String getMainConfigLocation() {
                 + CliConstants.MAIN_CONFIG_FILE_NAME;
     }
 
+    /**
+     * Returns location of the micro-gw.conf resource file.
+     *
+     * @return path configuration file
+     */
+    public static String getGWConfResourceLocation() {
+        return getCLIHome() + File.separator + CliConstants.GW_DIST_RESOURCES + File.separator
```

Comment:
Use `getResourceFolderLocation()` build the path

Reply:
fixed. thanks